CHANGELOG.md

```diff
@@ -2,8 +2,9 @@
 
 * Bumps Instabug Android SDK to `v11.8.0`
 * Bumps Instabug iOS SDK to `v11.7.0`
-* Adds `BugReporting.setDisclaimerText` API
-* Adds `BugReporting.setCommentMinimumCharacterCount` API
+* Adds new APIs:
+  1. `BugReporting.setDisclaimerText`
+  2. `BugReporting.setCommentMinimumCharacterCount`
 * Adds support for more locales:
   * czech
   * finnish
```

Comment:
**suggestion:** Locales should start with capital letter. We don't need to follow the same name used in code.